src/uu/mktemp/src/mktemp.rs

```diff
@@ -297,7 +298,7 @@ impl Params {
         let num_rand_chars = j - i;
 
         Ok(Self {
-            directory,
+            directory: directory.into(),
```

Comment:
Cool that we indeed already get the `PathBuf` from clap! However, we still need to fix this conversion from `String` inbetween, but I'll open an issue for that.